client/gutenberg/editor/main.jsx

```diff
@@ -32,14 +34,26 @@ class GutenbergEditor extends Component {
 	}
 
 	render() {
-		if ( isEmpty( this.props.siteSlug ) ) {
+		const { postType, siteId, siteSlug } = this.props;
+		if ( isEmpty( siteSlug ) ) {
 			return null;
 		}
 
-		overrideAPIPaths( this.props.siteSlug );
+		overrideAPIPaths( siteSlug );
```

Comment:
@vindl why is this here an not in `componentDidMount()`? do we really want the possibility that it could be called hundreds of times?

Reply:
Unintended leftover from original integration - it was easier to place it here initially to avoid calling it with an empty `siteSlug`.